CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+# Contributing
+
+Welcome to Tinted Theming, a community-driven project focused on
+creating and sharing themes. Our mission is to build a diverse
+collection of themes that can be easily adopted and customized. Whether
+you're looking to contribute your existing theme, join as a maintainer,
+help out with design, take part in discussions or report bugs, we're
+excited to have you on board.
+
+## Joining Tinted Theming
+
+Contributors can join the Tinted Theming Github organization in the
+following ways:
+
+1. Join an existing repository as one of the maintainers
+1. Transfer your repository into the Tinted Theming organization and
+   become a maintainer of that repository
+
+## Repository Transfer Process
+
+### Acceptance criteria
+
+Before transferring your repository, please ensure it meets the
+following criteria:
+
+- Follows our [Code of Conduct] and contribution guidelines
+- Aligns with Tinted Theming's thematic and aesthetic standards
+- Includes, at least basic, documentation on theme usage
+
+### Agreement
+
+When a repository is transferred to Tinted Theming, it then belongs to
+the Tinted Theming Github organization and community. The previous owner
+changes from being the owner of the repository to a maintainer of the
+repository. It is not mandatory to be a maintainer of transferring.
+
+Becoming a Tinted Theming maintainer when transfering in a repository is
+optional.
+
+### Process
+
+The process for transferring a repository into Tinted Theming is as
+follows:
+
+1. You will receive an invite to join the organization
+1. Accept the invite
+1. Go to the Settings page of the repository you want to transfer and
+   scroll to the bottom. You will see a "Danger Zone" section which
+   contains the option to "Transfer ownership".
+1. Click "Transfer ownership" and select Tinted Theming as the
+   organization you want to transfer to
+
+The steps on your side are now complete. Next steps an admin will take
+will be:
+
+1. Add you to a relevant Github team
+2. Add the team as the maintainer of the newly transfered repository
+3. If not already done, the primary branch will be set to `main`
+4. If the repository is a template repository, a PR will be created
```

Comment:
```suggestion
4. If the repository is a theme template repository, a PR will be created
```

A template repository is a github concept, and so it's a bit of an overloaded term. I'm open to other ways to phrase this.